packages/node_modules/@webex/internal-plugin-conversation/test/integration/spec/encryption.js

```diff
@@ -185,6 +185,24 @@ describe('plugin-conversation', () => {
           .then((c) => {
             assert.property(c, 'defaultActivityEncryptionKeyUrl');
           }));
+
+        it('sets the conversations\'s defaultActivityEncryptionKeyUrl to null', () => webex.internal.conversation.updateKey(conversation, {uri: null})
+          .then(() => checkov.webex.internal.conversation.get(conversation))
+          .then((c) => {
+            assert.isUndefined(c.defaultActivityEncryptionKeyUrl);
+            assert.property(c, 'kmsResourceObjectUrl');
+          }));
+
+        describe('rotate key', () => {
+          beforeEach(() => webex.internal.conversation.updateKey(conversation, {uri: null}));
```

Comment:
Could we move this into the `it` block below since there is only one test in the `describe` block?